### PR TITLE
Test/slack notification

### PR DESCRIPTION
**Pull request by MishaSavchuk14**

### CODE REVIEW

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+name: Basic test push
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:1]: 1

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:2]: 2

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+on:
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:3]: 3

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  pull_request:
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:4]: 4

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    types:
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:5]: 5

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+      - opened
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:6]: 6

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:7]: 7

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+jobs:
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:8]: 8

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+  pr-opened-slack-notifier:
```
Comment (misha-botscrew) [inline .github/workflows/pr-opened-slack-notifier.yml:9]: 9

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+    name: Send notification into slack when PR is opened
```
Comment (misha-botscrew) [inline .github/workflows/pr-opened-slack-notifier.yml:10]: 10

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+    runs-on: ubuntu-latest
```
Comment (misha-botscrew) [inline .github/workflows/pr-opened-slack-notifier.yml:11]: 11

**Review by misha-botscrew — APPROVED** [review]

**Review by misha-botscrew — APPROVED** [review]

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+    steps:
```
Comment (misha-botscrew) [inline .github/workflows/pr-opened-slack-notifier.yml:12]: 12

**Review by misha-botscrew — APPROVED** [review]

**Review by misha-botscrew — APPROVED** [review]

**Review by misha-botscrew — APPROVED** [review]

**Review by misha-botscrew — APPROVED** [review]

File: .github/workflows/pr-opened-slack-notifier.yml [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+      - uses: slackapi/slack-github-action@v1.18.0
```
Comment (MishaSavchuk14) [inline .github/workflows/pr-opened-slack-notifier.yml:13]: 13

**Pull request merged**